src/graph/transform/to_block.cc

```diff
@@ -13,17 +13,18 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  *
- * @file graph/transform/to_bipartite.cc
+ * @file graph/transform/to_block.cc
  * @brief Convert a graph to a bipartite-structured graph.
  */
 
```

Comment:
Add the following snippet here.
```
/**
 * Tested via python wrapper: python/dgl/path/to/to_block.py
 */
```

Reply:
If the test of cuda_to_block.cu if also covered by the py test, then add this snippet as well.

Reply:
added.